doc/nrf/known_issues.rst

```diff
@@ -827,18 +828,18 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
```

Comment:
We need to add another known issue:
```
KRKNWK-14748: Matter command timeout when a Matter device becomes a Thread router
  When a Full Thread Device becomes a router, it will ignore incoming packets for a short period of time, typically between 1-2 seconds.
  This may disrupt the communication over Matter and lead to transaction timeouts.
  
  **Note** In more recent versions of Matter, this problem has been eliminated by enhancing Matter's Message Reliability Protocol.
  This fix will be included in the future versions of NCS.
```